src/helper/CMakeLists.txt

```diff
@@ -32,6 +34,18 @@ endif()
 
 add_executable(sddm-helper ${HELPER_SOURCES})
 target_link_libraries(sddm-helper Qt5::Network Qt5::DBus Qt5::Qml)
+if(${CMAKE_SYSTEM} MATCHES "FreeBSD")
+    # On FreeBSD (possibly other BSDs as well), we want to use
```

Comment:
If this is true then you have to check also if CMAKE_SYSTEM_NAME equals to OpenBSD or NetBSD